app/templates/admin/administer-user-modal.jade

```diff
@@ -89,14 +91,17 @@ block modal-body-content
 
     else
       .form-group
-        label Number of Licenses
-        input#seats-input.form-control(type="number", name="maxRedeemers")
+        label
+          span.spr Number of Licenses
```

Comment:
Been moving away from the `.spr` and `.spl` classes because the space is not included when you copy the line. Better to use `=" "`.